drivers/usb/usbohci_new/usbohci.h

```diff
@@ -73,7 +78,11 @@ typedef struct _OHCI_HCD_ED {
   ULONG PhysicalAddress;
   ULONG Flags;
   LIST_ENTRY HcdEDLink;
+#ifdef _WIN64
+  ULONG Pad[6]; 
+#else
   ULONG Pad[8];
+#endif
```

Comment:
Out of interest, where does the change in padding come from? Did you dump the struct?